results/cmd/api/main_test.go

```diff
@@ -138,7 +140,13 @@ func TestUpdateTaskRun(t *testing.T) {
 		if err := proto.Unmarshal(taskrunblob, taskrun); err != nil {
 			t.Fatal("unmarshaling error: ", err)
 		}
-		if diff := cmp.Diff(taskrun.String(), r.GetTaskRun().String()); diff != "" {
+		if diff := cmp.Diff(taskrun.GetApiVersion(), r.GetTaskRun().GetApiVersion()); diff != "" {
+			t.Fatalf("Update Function not properly implemented: %v", diff)
+		}
+		if diff := cmp.Diff(taskrun.GetMetadata().GetUid(), r.GetTaskRun().GetMetadata().GetUid()); diff != "" {
+			t.Fatalf("Update Function not properly implemented: %v", diff)
+		}
+		if diff := cmp.Diff(taskrun.GetMetadata().GetName(), r.GetTaskRun().GetMetadata().GetName()); diff == "" {
```

Comment:
```suggestion
		if diff := cmp.Diff(taskrun, r.GetTaskRun()); diff != "" {
```

Reply:
The point of this test is to check uid and apiversion are changed because they are specified in fieldmask, but name is not because it is not in fieldmask

Reply:
done